pkg/internalapis/cnsoperator/cnsfilevolumeclient/v1alpha1/cnsfilevolumeclient_types.go

```diff
@@ -51,7 +51,3 @@ type CnsFileVolumeClientList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []CnsFileVolumeClient `json:"items"`
 }
-
-func init() {
```

Comment:
Can you clarify why are we touching existing APIs?

Reply:
Previously there was only `cnsfilevolumeclient` as an internal api. Now that we are adding `triggercsifullsync`, I have rearranged the internal apis in such a away they use same init and register for both apis. 

This is what we are following for external apis.